lib/active_model/serializer.rb

```diff
@@ -125,6 +125,10 @@ def json_key
       @root || object.class.model_name.to_s.underscore
     end
 
+    def fields(keys)
+      keys
+    end
```

Comment:
I'd like to narrow the Serializer interface rather than broaden it.  Every method we add has the potential to interfere with an attribute on the model.

Also, with respect to this change, couldn't you just use the key option?

```ruby
class FooSerializer < AM::Serializer
  attribute :name, key: title
end
```

```ruby
FooSerializer.new(foo).attributes
FooSerializer._attributes_keyss
FooSerializer._attributes
```